utils/utils.go

```diff
@@ -188,6 +196,27 @@ func IsPodmanInstalled() bool {
 	return true
 }
 
+// IsContainerdInstalled checks whether nerdctl and containerd is installed/running.
+func IsContainerdInstalled() bool {
+	if _, err := RunCmdAndWait("nerdctl", "info"); err != nil {
```

Comment:
Is nerdctl installation enough to check? Don't we need to check the actual containerd daemon?

Reply:
Yes, `nerdctl info` will check `containerd` daemon status, so I need to use `RunCmdAndWait` to wait for the output result. In my mac (using `colima`), this error occurs: `FATA[0000] colima is not running`; and linux will print like this: `FATA[0000] cannot access containerd socket "/run/containerd/containerd.sock": no such file or directory`